generators/server/templates/src/main/java/package/config/_CacheConfiguration.java

```diff
@@ -25,8 +25,8 @@
 import org.ehcache.expiry.Duration;
 import org.ehcache.expiry.Expirations;
 import org.ehcache.jsr107.Eh107Configuration;
-
 import java.util.concurrent.TimeUnit;
+import org.springframework.boot.autoconfigure.cache.JCacheManagerCustomizer;
```

Comment:
I think this is used for ehcache. I just changed the import order. I believe it was hanging separately in a if condition.